x-pack/plugins/cloud_security_posture/public/pages/findings/layout/findings_distribution_bar.tsx

```diff
@@ -64,14 +66,14 @@ const PassedFailedCounters = ({ passed, failed }: Pick<Props, 'passed' | 'failed
     >
       <Counter
         label={i18n.translate('xpack.csp.findings.distributionBar.totalPassedLabel', {
-          defaultMessage: 'Passed',
+          defaultMessage: 'Passed Findings',
```

Comment:
Should we delete the translations for this as we changed the value?

Reply:
looks like if CI passes then we don't. 

it won't pass if there's translations for an `id` we deleted